ros2_socketcan/launch/socket_can_receiver.launch.py

```diff
@@ -77,6 +78,7 @@ def generate_launch_description():
 
     return LaunchDescription([
         DeclareLaunchArgument('interface', default_value='can0'),
+        DeclareLaunchArgument('enable_can_fd', default_value='false'),
```

Comment:
[nits] Do we need the same argument in socket_can_bridge.launch.xml?

Reply:
Added. Thanks for the suggestion!